packages/tailwindcss/src/candidate.test.ts

```diff
@@ -935,6 +935,154 @@ it('should parse a utility with an implicit variable as the modifier', () => {
   `)
 })
 
+it('should properly decode escaped underlines but not underlines to spaces for CSS variables in arbitrary positions', () => {
```

Comment:
```suggestion
it('should properly decode escaped underscores but not convert underscores to spaces for CSS variables in arbitrary positions', () => {
```

Not 100% sure what you wanted to say exactly, but replaced `underlines` with `underscores` and added missing word (or at least it felt like something was missing?)

Reply:
hahaha yeah i think I wanted to say what you're saying 😆